### PR TITLE
Docsite updates

### PR DESCRIPTION
**Pull request by bdenham**
This PR consolidates the accumulation of the updates from PR #41 that are not related to the template files in that PR.

### CODE REVIEW

File: src/content/docs/dropins/all/installing.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: PDP Installation
+title: Installing dropins
```
Comment (jeff-matthews) [inline src/content/docs/dropins/all/installing.mdx:2]: The body content is still specific to PDP and needs to be genericized.

File: astro.config.mjs [hunk 2]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
         label: 'Dropins',
         items: [
           {
-            label: 'Overview',
-            link: '/dropins/'
+            label: 'Introduction',
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

File: astro.config.mjs [hunk 3]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
...
+            link: '/dropins/all/introduction/'
           },
           {
-            label: 'How to customize',
-            autogenerate: {
-              directory: '/dropins/customize/'
-            }
+            label: 'Anatomy of a dropin',
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

File: astro.config.mjs [hunk 4]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
...
+          {
+            label: 'Installing dropins',
+            link: '/dropins/all/installing/'
+          },
+          {
+            label: 'Branding dropins',
+            link: '/dropins/all/branding/'
+          },
+          {
+            label: 'Localizing dropins',
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

File: astro.config.mjs [hunk 5]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
...
+          {
+            label: 'Styling dropins',
+            link: '/dropins/all/styling/'
+          },
+          {
+            label: 'Extending dropins',
+            link: '/dropins/all/extending/'
+          },
+          {
+            label: 'Enriching dropins',
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

File: astro.config.mjs [hunk 6]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
...
           },
           {
             label: 'Product Details Page',
             collapsed: false,
             items: [
-              { label: 'Overview', link: '/dropins/product-details/' },
-              { label: 'PDP Installation', link: '/dropins/product-details/pdp-installation/' },
+              { label: 'PDP Anatomy', link: '/dropins/product-details/pdp-anatomy/' },
+              { label: 'PDP Configurations', link: '/dropins/product-details/pdp-configurations/' },
               { label: 'PDP Styles', link: '/dropins/product-details/pdp-styles/' },
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

File: astro.config.mjs [hunk 7]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
...
-              { label: 'PDP Containers', link: '/dropins/product-details/pdp-containers/' },
               { label: 'PDP Slots', link: '/dropins/product-details/pdp-slots/' },
-              { label: 'PDP Functions', link: '/dropins/product-details/pdp-functions/' }
+              { label: 'PDP Functions', link: '/dropins/product-details/pdp-functions/' },
+              { label: 'PDP Data & Events', link: '/dropins/product-details/pdp-data-events/' },
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

File: astro.config.mjs [hunk 8]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
...
+              { label: 'How to override styles', link: '/dropins/product-details/override-styles/' },
+              { label: 'How to change labels', link: '/dropins/product-details/change-labels/' },
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

File: astro.config.mjs [hunk 9]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
...
+              { label: 'How to override slots', link: '/dropins/product-details/override-slots/' },
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

File: astro.config.mjs [hunk 10]
```diff
@@ -90,25 +91,51 @@ export default defineConfig({
...
+              { label: 'How to add enrichment', link: '/dropins/product-details/add-enrichment/' },
```
Comment (jeff-matthews) [inline astro.config.mjs]: We shouldn't include links to empty files in the sidenav.

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
If the base branch was an integration branch and not `develop`, I wouldn't have flagged the links to empty topics in the sidenav.

If the goal is to merge with mainline and deploy to prod, we need to remove those links first. I'm ok including the empty files themselves, but it's still not really a best practice across our doc repos and should be avoided.

**Comment by bdenham** [comment]
@jeff-matthews Updated

File: astro.config.mjs [hunk 11]
```diff
@@ -89,26 +90,52 @@ export default defineConfig({
...
+          //   label: 'Enriching dropins',
+          //   link: '/dropins/all/enriching/'
+          // },
           {
             label: 'Product Details Page',
             collapsed: false,
             items: [
-              { label: 'Overview', link: '/dropins/product-details/' },
-              { label: 'PDP Installation', link: '/dropins/product-details/pdp-installation/' },
-              { label: 'PDP Styles', link: '/dropins/product-details/pdp-styles/' },
```
Comment (jeff-matthews) [inline astro.config.mjs:108]: The [PDP Styles](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/product-details/pdp-styles/) page looks like it has good content. I think we should keep it in the sidenav.

File: astro.config.mjs [hunk 12]
```diff
@@ -89,26 +90,52 @@ export default defineConfig({
...
+          // {
+          //   label: 'Enriching dropins',
+          //   link: '/dropins/all/enriching/'
+          // },
           {
             label: 'Product Details Page',
             collapsed: false,
             items: [
-              { label: 'Overview', link: '/dropins/product-details/' },
-              { label: 'PDP Installation', link: '/dropins/product-details/pdp-installation/' },
```
Comment (jeff-matthews) [inline astro.config.mjs:107]: I'm not sure hiding the [PDP Installation](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/product-details/pdp-installation/) page from the sidenav is the right move. At least not until we have a generic installation page under `dropins/all/installing.mdx`.

Also, the [Cart Installation](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/cart/cart-installation/) page is still displayed in the sidenav in this branch, which creates inconsistency.

**Review by jeff-matthews — COMMENTED** [review]
A couple more comments related to the sidenav.

I think this PR is still attempting to do some template-style restructuring in addition to updating components/styles. It's a little confusing exactly what we're trying to accomplish with this.

**Pull request closed without merging**